cypress/objects/LoginPage.js

```diff
@@ -0,0 +1,63 @@
+class LoginPage {
+    visit() {
+        cy.visit('/');
+    }
+
+    getUserValidationIndicator() {
+        return cy.get('.Select.not-valid');
+    }
+    getSubmitButton() {
+        return cy.get('[type="submit"]');
+    }
+
+    getSpecificUserFromDropDown(userName) {
+        return cy.get(`[aria-label="${userName}"]`)
+    }
+
+    getUserIdForm() {
+        return cy.get('[id="loginForm.userId"]')
+    }
+
+    getUserRoleForm() {
+        return cy.get('[id="loginForm.role"]')
+    }
+
+    getUserRoleFromDropDown(userRole) {
+        return cy.get(`[aria-label="${userRole}"]`)
+    }
+
+    url() {
+        return cy.url()
+    }
+
+    getPageTitle() {
+        return cy.get('.page__title')
+    }
+
+    getItemCalendar() {
+        return cy.get('.calendar')
+    }
+
+    getFormTitle() {
+        return cy.get('.tile.form')
+    }
+
+    getUserInfoTitle(userName) {
```

Comment:
Methods like these could be put in `common.js` or `pageComponents.js` files, because this is shared between pages and if you would want to check username or navigation bar it would look weird if you have to use `LoginPage `to do so.